doc/source/glossary.rst

```diff
@@ -2,6 +2,542 @@
 Glossary
 ********
 
-.. toctree::
+..  glossary::
 
-.. automodule:: numpy.doc.glossary
+    (`n`,)
+
+      A tuple with one element. The trailing comma distinguishes a one-element
+      tuple from a parenthesized ``n``.
+
+
+    -1
+
+      Used as a dimension entry, ``-1`` instructs NumPy to choose the length
+      that will keep the total number of elements the same.
+
+
+    ``...``
+
+      **When indexing an array**, shorthand that the missing axes, if they
+      exist, are full slices.
+
+        >>> a = np.arange(24).reshape(2,3,4)
+
+        >>> a[...].shape
+        (2, 3, 4)
+
+        >>> a[...,0].shape
+        (2, 3)
+
+        >>> a[0,...].shape
+        (3, 4)
+
+        >>> a[0,...,0].shape
+        (3,)
+
+      It can be used at most once:
+
+        >>> a[0,...,0,...].shape
```

Comment:
This should do the trick:
```python
>>> a[0,...,0,...].shape
Traceback (most recent call last):
  ...
IndexError: an index can only have a single ellipsis ('...')
```

Reply:
I threw up my hands in the latest commit, but this is good to know, thanks.

Reply:
For future reference: the official `doctest` documentation goes into a bit more detail about exceptions and doctesting ([docs](https://docs.python.org/3/library/doctest.html#what-about-exceptions)).

Reply:
I actually was on that page. I read "just paste in the traceback" in the first sentence and never reached the truthful part.  Gotta love that developer swagger. Now that I know, I'll have a serious look. Thanks.